src/helper/tw.ts

```diff
@@ -0,0 +1,11 @@
+// use this tagged template literal to enable formatting outside of className
+// works in vs code with this config:
+/*
+
+"tailwindCSS.experimental.classRegex": ["tw`([^`]*)"]
```

Comment:
does this mean everybody needs to add this? 
so we should probably put it in the wiki, right?

Reply:
very nice abstraction!

Reply:
good idea, I have added it to https://github.com/serlo/frontend/wiki/VS-Code